Exiled.API/Features/Roles/Scp106Role.cs

```diff
@@ -82,7 +82,7 @@ internal Scp106Role(Scp106GameRole baseRole)
         /// <summary>
         /// Gets the <see cref="Scp106Attack"/>.
         /// </summary>
-        public Scp106Attack Attack { get; }
+        public Scp106Attack AttackAbility { get; }
```

Comment:
There a breaking change here

Reply:
it makes conflict with the Attack Method

Reply:
that's why I changed it